packages/experiments/src/utilities/chicletHelper.tsx

```diff
@@ -0,0 +1,71 @@
+import * as React from 'react';
+import { Image } from 'office-ui-fabric-react/lib/Image';
+
+const ASSET_CDN_BASE_URL = 'https://static2.sharepointonline.com/files/fabric/assets';
+
+export function renderPreview(
+  imageUrl?: string,
+  imageHeight?: string,
+  imageWidth?: string,
+  imageAlt?: string
+): React.ReactElement<React.HTMLAttributes<HTMLDivElement>> | undefined {
+  let image;
+  if (imageUrl) {
+    image = <Image width={imageWidth} height={imageHeight} src={imageUrl} role="presentation" alt={imageAlt ? imageAlt : undefined} />;
+  }
+  return image;
+}
+
+export function renderIcon(
+  itemType?: string,
+  style?: string,
+  imageProvided?: boolean
+): React.ReactElement<React.HTMLAttributes<HTMLDivElement>> | undefined {
+  let src;
+  let icon;
+
+  if (itemType !== null && !imageProvided) {
```

Comment:
We can actually just do

```suggestion
  if (itemType && !imageProvided) {
```

The following values in TypeScript are considered falsey:
false, 0, "" (empty string), null, undefined, NaN